### PR TITLE
Fix commands and add ClearName feature

### PR DESCRIPTION
**Pull request by phoenixztt**
Change weird if else code to a more proper way of doing commands and add a counter/command feedback for radius command

Add ClearName Feature

Makes it so that if enabled, when you use a specific nametag name, it removes the custom name from a villager

### CODE REVIEW

**Comment by D-VR** [comment]
I like the clearNametag functionality, would it make sense to not have a specific name but instead have it be a boolean config option which clears the name if a villager gets enabled again?

**Comment by D-VR** [comment]
I can do this change + add other small fixes to this PR or a later PR since i have time again today 

(if y'all agree to the boolean config option)

**Comment by phoenixztt** [comment]
> I like the clearNametag functionality, would it make sense to not have a specific name but instead have it be a boolean config option which clears the name if a villager gets enabled again?

Maybe even using PDC to store the old name a restore it to that

**Comment by D-VR** [comment]
hmh.. yes would need to add setter getter and has for it, not too bad though

File: src/main/java/rebelmythik/antivillagerlag/commands/RadiusOptimizeCommand.java [hunk 1]
```diff
@@ -80,12 +79,14 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
                             // set all necessary flags and timers
                             VillagerUtilities.setMarker((Villager) vil, plugin);
                             VillagerUtilities.setNewCooldown((Villager) vil, plugin, cooldown);
+                            count++;
                         }
 
                     }
                 }
+                sender.sendMessage(colorcodes.cm("&aOptimized "+count+" villagers!"));
```
Comment (RebelMythik) [inline src/main/java/rebelmythik/antivillagerlag/commands/RadiusOptimizeCommand.java:87]: Make the message not hard coded

File: src/main/java/rebelmythik/antivillagerlag/AntiVillagerLag.java [hunk 2]
```diff
@@ -22,8 +25,15 @@ public final class AntiVillagerLag extends JavaPlugin {
     public void onEnable() {
         // Plugin startup logic
         this.getServer().getPluginManager().registerEvents(new EventListenerHandler(this),this);
+        ColorCode colorcodes = new ColorCode();
```
Comment (RebelMythik) [inline src/main/java/rebelmythik/antivillagerlag/AntiVillagerLag.java:28]: Not going to set permissions here. They are already in plugin.yml

**Review by RebelMythik — CHANGES_REQUESTED** [review]
will pull when its fixed

**Pull request closed without merging**